app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -306,3 +306,22 @@ input:checked + .slider::before {
     justify-content: flex-end;
     margin-right: 15px;
 }
+
+
+.icon-message {
+    opacity: 0;
+    color: $color-text-blue-dark;
+}
+
+.icon-message:hover {
```

Comment:
It moves a little as board shrinks, so fix and throw in a little fade in animation